galsim/phase_psf.py

```
@@ -0,0 +1,1017 @@
+                            resulting PSF.  [Default: 0.0*galsim.arcmin]
+    @param scale_unit       Scale of output profile.  [Default: galsim.arcsec]
+    @param diam             Diameter in meters of aperture used to compute PSF from phases.
+                            [Default: 8.4]
+    @param obscuration      Linear dimension of central obscuration as fraction of pupil linear
+                            dimension, [0., 1.).  [Default: 0]
+    @param interpolant      Either an Interpolant instance or a string indicating which interpolant
+                            should be used.  Options are 'nearest', 'sinc', 'linear', 'cubic',
+                            'quintic', or 'lanczosN' where N should be the integer order to use.
+                            [default: galsim.Quintic()]
+    @param pad_factor       Additional multiple by which to zero-pad the PSF image to avoid folding
+                            compared to what would be employed for a simple Airy.  Note that
+                            `pad_factor` may need to be increased for stronger aberrations, i.e.
+                            those larger than order unity.  [default: 1.5]
+    @param oversampling     Optional oversampling factor for the wrapped InterpolatedImage. Setting
+                            `oversampling < 1` will produce aliasing in the PSF (not good).
+                            Usually `oversampling` should be somewhat larger than 1.  1.5 is
+                            usually a safe choice.  [default: 1.5]
+    @param gsparams         An optional GSParams argument.  See the docstring for GSParams for
+                            details. [default: None]
+    """
+    def __init__(self, screen_list, lam=500., exptime=0.0, flux=1.0,
+                 theta_x=0.0*galsim.arcmin, theta_y=0.0*galsim.arcmin,
+                 scale_unit=galsim.arcsec, interpolant=None,
+                 diam=8.4, obscuration=0.6,
```

Oops.  You claimed above that the default obscuration is 0.  Here you have 0.6.